swiftwinrt/Resources/Support/Swift+Extensions.swift

```diff
@@ -58,6 +101,19 @@ extension Character {
   }
 }
 
+@_spi(WinRTInternal)
+extension Character: WinRTBridgeable {
```

Comment:
`Swift.Character` is not an appropriate projection for `WCHAR` because the Swift -> ABI conversion is very lossy (a Swift character is closer to a string as it supports decomposed diacritics and such).

`Swift.Unicode.UTF16.CodeUnit` (aka UInt16) is an appropriate type, but it breaks overloading between UInt16 and WChar in projections so in my generator I define a `WideChar` struct in Swift.

Reply:
agreed this needs to be fixed but that's a different problem